src/raster/r.in.nasadem/r.in.nasadem.py

```diff
@@ -301,9 +301,8 @@ def download_tile(tile, url, pid, version, username, password):
         time.sleep(0.5)
     except urllib.error.URLError as err:
         grass.fatal(
-            _(
-                f"Download of tile {local_tile} from URL {remote_tile} was not "
-                f"successful:\n{err}"
+            _("Download of tile {0} from URL {1} was not successful:\n{2}").format(
```

Comment:
Cosmetic change: Ordering string interpolation is no needed

`"Download of tile {} from URL {} was not successful:\n{}"` work as expected

but better if number of string interpolation > 1 (your case is 3), use variable substitutions by name inside `{}`

```
_("Download of tile {local_tile} from URL {remote_tile} was not successful:\n{err}").format(
    local_tile=local_tile, remote_tile=remote_tile, err=err,
)
```